_rules/inline-link-in-paragraph-has-distinguishable-style-not-based-on-color-alone-88407d.md

```diff
@@ -27,16 +27,17 @@ This rule applies to any [visible][] HTML element that is a [semantic link][], f
 
 For each test target, at least one of the following is true:
 
-- **distinguishing style**: the element has a [distinguishing style][] not based on color alone from the **non-link line text** elements; or
-- **distinguishing content**: element has content (such as an image or text), located in the proximity of the element (i.e. there is no other content visually in between this content and the target element), that indicates the test target is a link.
+- **distinguishing style**: the element has a [distinguishable style][] not based on color alone from the **non-link line text** elements; or
```

Comment:
What if the distinguishing style is applied to an ancestor?
`<p>Lorem <span class="border"><span><span><a href="#" class="no-style">link</a></span></span></span> Ipsum</p>`
is currently failing the rule but passing the SC.
And the latest assumption is accepting styling on ancestors, so doesn't catch that.

Reply:
The only way I see to tackle this is to extend the assumption to include ancestors...